rust-toolchain

```diff
@@ -1 +1 @@
-nightly
+nightly-2020-11-01
```

Comment:
I don't think the GitHub Action is respecting this, but it shouldn't be too hard to read from this file to do that (cf. me [doing the same for Node](https://github.com/signalapp/libsignal-client/pull/20/files#diff-4e88a8cc63d2540befa3412886ac9b9db5cd10bce852960298517ea4bd7f7bb7R26-R31)).